### PR TITLE
Murder Player Feature

### PR DESCRIPTION
**Pull request by manianac**
Supports murdering as an imposter
Kill cooldown is respected, button is hidden until kills are allowed

### CODE REVIEW

**Review by OsOmE1 — CHANGES_REQUESTED** [review]
Change the way you get the kill cooldown

**Comment by manianac** [comment]
Updated to use LocalPlayer::KillTimer

**Review by manianac — COMMENTED** [review]
Updated to use killtimer

**Review by RealMVC — APPROVED** [review]

File: appdata/il2cpp-translations.h [hunk 1]
```diff
@@ -45,6 +45,7 @@ const std::vector<KLASS_PAIR> KLASS_TRANSLATIONS = {
 	{ {"", "HudOverrideTask"}, {"", "LFOILEODBMA"} },
 	{ {"", "ImportantTextTask"}, {"", "ABFBCNBODMA"} },
 	{ {"", "KeyboardJoystick"}, {"", "ADEHDODPMHJ"} },
+	{ {"", "KillButtonManager"}, {"", "MLPJGKEACMM"} },
```
Comment (OsOmE1) [inline appdata/il2cpp-translations.h:48]: Remove this translation because you don't use it.

**Review by OsOmE1 — CHANGES_REQUESTED** [review]

**Review by OsOmE1 — APPROVED** [review]

**Pull request merged**